articles/quickstart/native/android/03-session-handling.md

```diff
@@ -140,30 +152,64 @@ The `SecureCredentialsManager` can prompt the user for local device authenticati
 
 ## Log the User Out
 
-To log the user out, you remove their credentials and navigate them to the login screen. When using a Credentials Manager you do that calling `clearCredentials`.
+To log the user out, it is normally enough to remove their credentials and navigate them back to the login screen. When using a Credentials Manager you do that calling `clearCredentials`. In addition, you could ask the `WebAuthProvider` to remove the cookie set by the Browser at authentication time, so that the users are forced to re-enter their credentials the next time they try to authenticate. The sample combines these two strategies.
```

Comment:
```suggestion
To log the user out, it is normally enough to remove their credentials and navigate them back to the login screen. When using a Credentials Manager, you do that calling `clearCredentials`. In addition, you can ask the `WebAuthProvider` to remove the cookie set by the browser at authentication time. This will force users to re-enter their credentials the next time they try to authenticate. The sample below combines these two strategies.
```